### ISSUE TITLE
Organization to contain software packages?

### ISSUE THREAD

**Issue by rlopezcoto** [juanep97/iop4#47]
Hi, since this software is meant to be published in a refereed publication and so on, it always looks better if it is contained in a "permanent" organization instead of a personal github repo.
I suggest that we can create the "vhega" organization and migrate libraries like this one (maybe also [MUTIS](https://github.com/juanep97/MUTIS)) The migration should be transparent for owners/admins and so on, but it will just be contained under a different URL that will make it look more permanent.

**Comment by morcuended** [comment]
I agree. MUTIS for example is already under https://github.com/IAA-CSIC organization. Would that work?

**Comment by juanep97** [comment]
I agree, @rlopezcoto, can you create the VHEGA organization? I have to check what happens with the zenodo DOI.

**Comment by juanep97** [comment]
I think we should have a VHEGA organization, independently of whether IOP4 goes in it or in the IAA-CSIC one. It also needs to be an organization where I can create repositories, for GitHub to be able to transfer (so we would have to ask IAA-CSIC).

**Comment by rlopezcoto** [comment]
do you who is the admin of the IAA-CSIC organization?

**Comment by juanep97** [comment]
Probably [@bultakoo](https://github.com/Bultako).
